backend/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "watercool-backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.ts",
+  "scripts": {
+    "test": "npm run test",
+    "start": "tsc && node server.js"
```

Comment:
we can drop the tsc here for sure (I have seen typescript being used directly, without compiling to js first, for development purposes)
googling a bit - maybe ts-node would be useful for his

and then add script "build" that actually builds .ts into .js files 
thanks to that, we will get rid of these generated .js files in repository